src/java.desktop/share/classes/javax/swing/JFileChooser.java

```diff
@@ -1412,7 +1412,8 @@ public boolean isMultiSelectionEnabled() {
 
     /**
      * Returns true if hidden files are not shown in the file chooser;
-     * otherwise, returns false.
+     * otherwise, returns false. The default value of this property may be
+     * derived from the underlying Operating System.
```

Comment:
Should true and false be marked up as `{@code true}` and `{@code false}`?
Probably not, for consistency with other methods in the JFileChooser class.

Reply:
@aivanov-jdk Yeah, could have done that, but still lot of places in the same file doesn't have the consistency, so taking it up in another PR will be better I guess....